ffcx/codegeneration/definitions.py

```diff
@@ -114,6 +107,11 @@ def coefficient(self, t, mt, tabledata, quadrature_rule, access):
         iq_symbol = self.symbols.quadrature_loop_index
         ic_symbol = self.symbols.coefficient_dof_sum_index
 
+        if isinstance(access, str):
```

Comment:
Contradicts line 101: `access` is always `str` ?

Reply:
Good point. It's not uniform, from expression generator it's a symbol, and from integral generator it's a string.
I'll make it consistent.